src/extra/desktop.win.cpp

```diff
@@ -664,7 +664,7 @@ struct SendKeyEvent : public SendKeyEventBase {
       }
 
       auto lparam = MapVirtualKey(vkCode, MAPVK_VK_TO_VSC) << 16;
-      lparam = lparam | (state == 0) ? 0x00000001 : 0xC0000001;
+      lparam = lparam | (state == 0 ? 0x00000001 : 0xC0000001);
```

Comment:
Is this some auto refactor? They look both suspicious :D

Reply:
I believe you mean `lparam | (state == 0 ? 0x00000001 : 0xC0000001)`
Clang was giving a warning that | takes precedence over ? so it would evaluate to
`(lparam | state == 0) ? 0x00000001 : 0xC0000001`